recipe/meta.yaml

```diff
@@ -32,7 +32,10 @@ test:
   imports:
     - icu
   commands:
+    - pip check
     - python -c "from icu import Locale;locale = Locale('pt_BR')"
```

Comment:
is there a way to add this to `test.py`? I think you can add a `files:` section to the test section and commit these tests in a separate file.

Reply:
I don't think it is worth converting this single line test to a separate file.